buf.gen.yaml

```diff
@@ -0,0 +1,18 @@
+# Generate from proto-public in consul.
+#
+#   buf generate --template buf.gen.yaml ../consul/proto-public/ \
+#     --exclude-path ../consul/proto-public/pbconnectca
```

Comment:
Am I using `buf` right here?

Reply:
Seems legit to me! 😅 Could this be a make target, though?

Reply:
Yep, added `make proto-gen` for this.